playwright/pages/draftOrdersPage.ts

```diff
@@ -60,6 +60,14 @@ export class DraftOrdersPage extends BasePage {
   }
 
   async clickAddShippingCarrierButton() {
+    await this.page.waitForLoadState("networkidle");
```

Comment:
https://playwright.dev/docs/api/class-page#page-wait-for-load-state

Especially:
> Most of the time, this method is not needed because Playwright [auto-waits before every action](https://playwright.dev/docs/actionability).

and:
> 'networkidle' - DISCOURAGED wait until there are no network connections for at least 500 ms. Don't use this method for testing, rely on web assertions to assess readiness instead.